src/common/extend/disableDefaultStylePicker.js

```diff
@@ -0,0 +1,42 @@
+import { hooks } from 'wp';
+
+const { addFilter } = hooks;
+
+// Disable defaultStylePicker on the following blocks
+const disableOnBlocks = [
```

Comment:
Fine for me to have a static list here, because we won't have such a "dynamic" here...

Reply:
yes I assume too, that we won't have to much dynamic here. In the case we add some styles to our custom blocks, we need to expand the list here. For the amount of the blocks we have, won't be a big deal.